libs/s3/s3_common_test_lib.py

```diff
@@ -439,6 +434,13 @@ def validate_copy_content(src_bucket, src_object, dest_bucket, dest_object, **kw
     destchecksum = calculate_checksum(down_path2)
     assert_utils.assert_equal(srcchecksum, destchecksum, "Checksum match failed.")
     LOG.info("Validated content of copy object")
```

Comment:
this line should be at the end , i.e. after line 443

Reply:
incorporated this change